src/inverted_index.c

```diff
@@ -1377,36 +1372,8 @@ int IndexBlock_Repair(IndexBlock *blk, DocTable *dt, IndexFlags flags, IndexRepa
     Buffer_ShrinkToSize(&blk->buf);
   }
 
-  params->bytesAfterFix = blk->buf.offset;
+  params->bytesAfterFix = blk->buf.cap;
 
   IndexResult_Free(res);
   return frags;
 }
-
-int InvertedIndex_Repair(InvertedIndex *idx, DocTable *dt, uint32_t startBlock,
```

Comment:
It was deleted. It is not used, and the commit message when it was created (6 years ago) is:
"Allow for callback to inverted index repair function
This is not actually used by anything, but it may be helpful in the
future.".

Should I restore it?